index.js

```diff
@@ -26,6 +26,9 @@ export default e => {
   const NpcPlayer = useNpcPlayerInternal();
   const localPlayer = useLocalPlayer();
   const physics = usePhysics();
+  const PathFinder = usePathFinder();
+  const pathFinder = new PathFinder({voxelHeight: 1.5});
```

Comment:
Shouldn't this have a range of voxels as well?

Reply:
IMO

```
new PathFinder({voxelHeight: 1.5});
```

should be accepting a range.

If there is no range limit control, then it is possible this algorithm will run to infinity for long distances. And if there is a limit it should not be arbitrary, it should be listed here as an argument.

Reply:
I use maxIterStep to limit the A* algorithm's max steps.
https://github.com/webaverse/app/pull/2298/files#diff-6a50a9c285e09666c1bfc3816458b22f3e90b9e124bb5b08b02cdfcbf85ff057R28
And one A* step can create up to 4 voxels.

Reply:
Ok! I'll add.